### PR TITLE
Add redirects for add-on permission page

### PR DESCRIPTION
**Pull request by jobisoft**

### CODE REVIEW

File: vars/apache.yml [hunk 1]
```diff
@@ -94,6 +94,8 @@ apache_vhosts:
       RewriteEngine On
       RewriteCond %{HTTP:X-Forwarded-Proto} ^http$
       RewriteRule ^.*$ https://%{SERVER_NAME}%{REQUEST_URI} [L,R=301,NE]
+      RewriteCond %{REQUEST_URI} ^/([^/]+)/thunderbird/([^/]+)/extension-permissions$
+      RewriteRule (.*) https://support.mozilla.org/%1/kb/permission-request-messages-thunderbird-extensions [L]
```
Comment (Sancus) [inline vars/apache.yml:98]: Forcing locale like this is not generally recommended, as the list of supported languages in Thunderbird and SUMO are not necessarily the same and that can result in 404s. It's best to leave language to browser Accept-Language and SUMO negotiation, IMO.

Additionally, why is build id in this URL on the client side at all? Seems like a completely useless parameter.

Comment (jobisoft) [reply vars/apache.yml]: > Additionally, why is build id in this URL on the client side at all? Seems like a completely useless parameter.

I do not know, you have to ask Mozilla about that. Noting I can change (besides ignoring it).

> Forcing locale like this is not generally recommended, as the list of supported languages in Thunderbird and SUMO are not necessarily the same and that can result in 404s. It's best to leave language to browser Accept-Language and SUMO negotiation, IMO.

By trial-and-error I learned that I can leave out the locale and SUMO picks one. Is that what you prefer? I updated the pull request. If this is not what you had in mind, I could need a little push in the right direction.

Comment (Sancus) [reply vars/apache.yml]: > I do not know, you have to ask Mozilla about that. Noting I can change (besides ignoring it).

Hmm? According to the bug it's clearly set in our code: https://searchfox.org/comm-central/search?q=app.support.baseURL&path=profile%2F*.js&case=false&regexp=false

app.support.baseURL is set in `mail/app/profile/all-thunderbird.js`. If you look at the Firefox version they do `/%VERSION%/%OS%/%LOCALE%/` which is a lot more useful, though I'm not sure if there's a way to specify OS to SUMO. Still though, one could conceivably use those variables for something at least.

Anyway it's not critical but we should definitely change that.

> By trial-and-error I learned that I can leave out the locale and SUMO picks one. Is that what you prefer? I updated the pull request. If this is not what you had in mind, I could need a little push in the right direction.

Yes, that's what I meant. Sorry, should have been more specific, I forgot that you might not have context on how SUMO and most Mozilla websites work. In just about all cases, our sites have language negotiation that checks the Accept-Languages header and then redirects to the write locale, but there isn't a consistent supported languages list across all apps and sites, so it's best to always link language-agnostic, which you generally do by just leaving the language code out entirely. It works that way on [ATN as well, for example](https://addons.thunderbird.net/thunderbird/addon/gmail-conversation-view/), on mozilla.org and thunderbird.net.

Comment (jobisoft) [reply vars/apache.yml]: > Hmm? According to the bug it's clearly set in our code: https://searchfox.org/comm-central/search?q=app.support.baseURL&path=profile%2F*.js&case=false&regexp=false

I was dead sure it was toolkit code ... but I did link to that same code in bug 1623575 by myself. Sometimes this just happens. Sorry for the mixup.

I will request changing the URL. Closing this PR for now.

**Comment by jobisoft** [comment]
Should we keep the locale in the URL generated by the client? You said the client locale should never be used, so we could just get rid of it as well. The new format could look like so:
https://support.thunderbird.net/%APP%/%VERSION%/%OS%/

which produces URLs like this:
https://support.thunderbird.net/thunderbird/86.0a1/WINNT/extension-permissions

I keep the first` /thunderbird/` in the path to identify URLs generated by Thunderbird. Firefox is using just `/1/` here. I have no preference here.

Do you have any wishes you would like to see in the new format?

**Comment by Sancus** [comment]
I think we can keep the locale in, it's possible we may use it in the future(like if we end up with a Thunderbird-specific support site where we COULD make sure the languages are identical), and it's painful to change client URLs since it can take up to a year before the change reaches all users on a supported version.

I prefer /thunderbird/ to /1/ lol, it's a bit more meaningful.

I think `https://support.thunderbird.net/thunderbird/%VERSION%/%OS%/%LOCALE%/` is fine.

**Comment by jobisoft** [comment]
I have updated the pull request for the new URL layout, which landed in todays daily:
https://support.thunderbird.net/thunderbird/86.0a1/WINNT/en-US/extension-permissions

It would be great if you could approve this pull request.

**Comment by Sancus** [comment]
This is live now.

**Pull request merged**